edge-patron/src/main/resources/domain/edge-patron/features/patrons.feature

```diff
@@ -25,7 +25,7 @@ Feature: patron tests
     * call read('classpath:domain/edge-patron/features/util/initData.feature@PostOwnerAndCharges')
 
     Given url edgeUrl
-    And path 'patron/account/' + extSystemId
+    And path 'patron/account/' + extSystemId + '?includeCharges=true'
```

Comment:
Why has this request changed?

Reply:
That was a  mistake. its been changed now.